test/OpenFeature.Contrib.Providers.Flagd.Test/StringEvaluatorTest.cs

```diff
@@ -0,0 +1,78 @@
+using System.Collections.Generic;
+using JsonLogic.Net;
+using Newtonsoft.Json.Linq;
+using OpenFeature.Contrib.Providers.Flagd.Resolver.InProcess.CustomEvaluators;
+using Xunit;
+
+namespace OpenFeature.Contrib.Providers.Flagd.Test
+{
+    public class StringEvaluatorTest
```

Comment:
Do you need tests (or special handling) for non-string arguments (the spec dictates that non-strings should result in `false`).

See: https://flagd.dev/reference/specifications/custom-operations/string-comparison-operation-spec/